internal/pkg/agent/application/managed_mode.go

```diff
@@ -194,12 +197,35 @@ func (m *managedConfigManager) Run(ctx context.Context) error {
 		}
 	})
 
+	go func() {
+		for {
+			select {
+			case <-ctx.Done():
+				return
+			case err := <-actionDispatcher.Errors():
+				m.errCh <- err // err is one or more failures from dispatching an action
+			}
+		}
+	}()
```

Comment:
You should just start this before restoring the policy from disk. Then this gofunction will handle reading from the `errCh` and sending it to the `errCh` for the managed mode.